### PR TITLE
Adjust print statement in check_space

### PR DESCRIPTION
**Pull request by souravsingh**
- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make clean diff-cover` If it introduces new functionality in
  `scripts/` is it tested?
- [x] `make format diff_pylint_report cppcheck doc pydocstyle` Is it well
  formatted?
- [ ] Did it change the command-line interface? Only backwards-compatible
  additions are allowed without a major version increment. Changing file
  formats also requires a major version number increment.
- [ ] For substantial changes or changes to the command-line interface, is it
  documented in `CHANGELOG.md`? See [keepachangelog](http://keepachangelog.com/)
  for more details.
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?
- [x] Do the changes respect streaming IO? (Are they
  tested for streaming IO?)

### CODE REVIEW

**Comment by souravsingh** [comment]
@standage The failures due to docker are still happening.

File: khmer/kfile.py [hunk 1]
```diff
@@ -134,19 +134,19 @@ def check_space(in_files, force, _testhook_free_space=None):
...
+        message = 'Not enough free space on disk for output files;'
+        message += '\n       Need at least {:s} more.'.format(mem_needed)
+        message += '\n       Estimated Output size: {:s}'.format(mem_requested)
+        message += '\n       Free space: {:s}'.format(mem_available)
+        if force:
+            print('WARNING:', message, file=sys.stderr)
+        else:
+            message = 'ERROR: ' + message + \
+                '\nNOTE: This can be overridden using the --force argument'
+            raise SystemExit(message)
```
Comment (betatim) [inline khmer/kfile.py]: add another empty line here. PEP8 says you should have two empty lines between functions at the file level.

**Comment by betatim** [comment]
I'm thinking about the docker part that is failing. Sorry we don't have our CI system in order ;)

**Comment by souravsingh** [comment]
@betatim The tests are still failing even after merging the latest patch.

**Comment by standage** [comment]
@souravsingh It looks like the tests are passing now! The failure is due to a style code violation. [Line 4223](https://travis-ci.org/dib-lab/khmer/jobs/311191108#L4223) of the latest build shows this message.

```
$ make pep8
pep8 setup.py khmer/*.py scripts/*.py tests/*.py oxli/*.py
khmer/kfile.py:151:1: E302 expected 2 blank lines, found 1
make: *** [pep8] Error 1


The command "make pep8" exited with 2.
```

If you run `pep8` before you commit changes, that should help avoid any CI failures due to style.

Thanks!

**Comment by souravsingh** [comment]
Ah, I merely skipped through the full CI output and didn't notice the lint error. My bad!

**Pull request merged**